modules/ROOT/pages/event-tracking.adoc

```diff
@@ -236,13 +248,14 @@ services:
             replayLevel: ALL
 ----
 
-=== Configuring Via the Agent API During Runtime
+=== Configure Tracking Using the Agent API During Runtime
+
+The agent API enables you to change the agent tracking configuration during runtime.
```

Comment:
Can we add more context? Why/under what circumstances does a user want or need to do this?

Should these be task topics instead of concepts?